lib/ansible/modules/web_infrastructure/ansible_tower/tower_project.py

```diff
@@ -210,11 +210,13 @@ def main():
             if state == 'present':
                 org_res = tower_cli.get_resource('organization')
                 org = org_res.get(name=organization)
+                cred_res = tower_cli.get_resource('credential')
```

Comment:
I would like to capture the exeception better in the case where the credential is not found. Right now, that would be handled in the `NotFound` except block below, but the error message would report it as `organization not found` which would mislead the user.

I think likely you can just remove the `organization not found` line from 227. I believe that Tower CLI in its own exception info gives the proper details.